### DISCUSSION TITLE
Unnecessary `:where()` in build

### DISCUSSION THREAD

**Discussion by Steffan153** [tailwindlabs/tailwindcss#11115] [Ideas]
I mostly see this with the typography plugin. Example (after formatting the file):
```
abbr:where([title]) {
  -webkit-text-decoration: underline dotted;
  text-decoration: underline dotted;
}

.prose :where(p):not(:where([class~='not-prose'] *)) {
  margin-bottom: 1.25em;
  margin-top: 1.25em;
}
```

All the `:where()`s there can surely just be removed. Wouldn't `abbr[title]` and `.prose p:not([class~='not-prose'] *)` work just fine? This would reduce my build size by a lot.

**Comment by wongjn** [comment]
The `:where()` is to keep the [selector specificities](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) equal to or below `(0, 1, 0)`, so that utility classes will still be able to take precedence over these rules.

**Comment by Steffan153** [comment]
I see. But why not do `:where(.prose p:not([class~='not-prose'] *))` which is shorter? Is `0,0,0` a too low specifity?

Or `.prose :where(p:not([class~='not-prose'] *))`, which is still `0,1,0`.
